certified-connectors/Iconfirm/readme.md

```diff
@@ -0,0 +1,106 @@
+# Iconfirm Connector
+Iconfirm offers a suite of tools and functionality to help organizations comply with the requirements set by GDPR with additional features to ease the day to day documentation and insight for your organization.
+
```

Comment:
@iconfirm-development Please refer the template provided for readme.md file and add the required sections.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Readme was set up with the guide from January when the connector was submitted, but i've gone over it again and updated with the new information now